package-lock.json

```diff
@@ -9,7 +9,7 @@
       "version": "0.1.0",
       "dependencies": {
         "@fluentui/react": "^7.168.2",
-        "@fruk/simulator-core": "^1.8.9",
+        "@fruk/simulator-core": "^1.8.10",
```

Comment:
We should pin the SimulatorCore to make sure those kind of changes do not happen